mkdocs.yml

```
@@ -76,6 +76,7 @@ nav:
           - Curves: references/curves.md
           - Custom Materials: references/custom_materials.md
           - Custom Properties: references/custom_properties.md
+          - Damageable Object: references/damageable_object.md
```

```suggestion
          - Damageable Objects: references/damageable_object.md
```

I think we use plural everywhere